tests/dummy/config/environment.js

```diff
@@ -18,6 +18,10 @@ module.exports = function(environment) {
       // when it is created
     },
 
+    'ember-metrics': {
```

Comment:
Forces the inclusion of segment and google-tag-manager since the unit tests use them.. also proves the feature works :)